data/inventory/inventory.go

```diff
@@ -24,8 +35,11 @@ func (i Inventory) MarshalJSON() ([]byte, error) {
 }
 
 // SetItem stores a value into the inventory data structure.
```

Comment:
Having the length limit in the comment could be also useful when you're reading the documentation.

Reply:
512 is not the limit, what is it?

Reply:
Set to 425

Reply:
Sorry, what I meant in my comment was that having the info about the limit in the comment was useful. Reading the documentation about the function you could see the limit value, otherwise you would need to dig into the source code.